content/docs/configuration/annotations.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Annotations
```

Comment:
@jsoref is there a way to make sure this page is kept up to date?
If not, could you add a comment here (for developers of the website) to explain the purpose of this page and how we should keep it up to date?

Reply:
Or is this what you wanted?
https://github.com/cert-manager/website/blob/22ffc2b85a6278f85bc5721da585e7601b574517/content/docs/configuration/annotations.md?plain=1#L4-L6